src/components/HTMLEngineProvider/HTMLRenderers/ImageRenderer.tsx

```diff
@@ -61,17 +72,20 @@ function ImageRenderer(props) {
         <ShowContextMenuContext.Consumer>
             {({anchor, report, action, checkIfContextMenuActive}) => (
                 <PressableWithoutFocus
-                    style={[styles.noOutline]}
+                    style={[styles.noOutline as StyleProp<ViewStyle>]}
```

Comment:
```suggestion
                    style={styles.noOutline}
```
this should work

Reply:
Update this type
```
type AddOutlineWidth = <TStyle extends TextStyle | ViewStyle>(theme: ThemeColors, obj: TStyle, val?: number, hasError?: boolean) => TStyle;
```
and get rid of assertion